### DISCUSSION TITLE
关于版本的区别和选择

### DISCUSSION THREAD

**Discussion by Deecyn** [volantis-x/hexo-theme-volantis#655] [Q&A] [answered]
有个困惑请教一下，看着 volantis 主题有很多大版本的，文档也是，从 2.X 到 5.X。每个大版本主要的区别是啥？我们该如何选择吗，还是选择最新的稳定版就好？如果可以的话，建议加个说明。

**Comment by xaoxuu** [comment, marked as answer]
建议用最新版的
